OwenConferenciasApp/screens/NewEvents.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  ScrollView,
+  TextInput,
+  Alert,
+} from 'react-native';
+
+export default class NewEvent extends React.Component {
+  render() {
+    return (
+      <View style={styles.mainContainer}>
+        <ScrollView style={styles.scrollView}>
+          <View style={styles.cardEvent}>
+            <Text style={styles.txtTitle}>New conference</Text>
+            <View style={styles.formNewEvent}>
+              <TextInput
+                style={styles.inputEvents}
+                placeholder="Name of the conference:"
+              />
+              <TextInput style={styles.inputEvents} placeholder="Topic:" />
+              <TouchableOpacity
+                style={styles.btnDayHour}
+                onPress={this.showPicker}>
+                <Text style={styles.txtButtons}>Select the day</Text>
+              </TouchableOpacity>
+              <TouchableOpacity style={styles.btnDayHour}>
+                <Text style={styles.txtButtons}>Select the hour</Text>
+              </TouchableOpacity>
+              <TouchableOpacity
+                style={styles.btnAddConf}
+                onPress={() =>
+                  Alert.alert('Are you sure ta add this conference')
+                }>
+                <Text style={styles.txtButtons}>Add conference</Text>
+              </TouchableOpacity>
+            </View>
+          </View>
+        </ScrollView>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    height: '100%',
+    backgroundColor: '#F0F1F6',
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  scrollView: {
+    height: '100%',
+  },
+  txtTitle: {
+    fontSize: 30,
+    marginTop: 20,
+    marginLeft: 15,
+  },
+  cardEvent: {
+    width: '85%',
+    height: 500,
+    marginTop: 50,
+    marginRight: 'auto',
+    marginLeft: 'auto',
+    backgroundColor: '#ffff',
+    borderRadius: 20,
+  },
+  formNewEvent: {
+    marginTop: 30,
+    marginLeft: 15,
+    marginRight: 15,
+  },
+  inputEvents: {
+    width: '90%',
+    height: 50,
+    fontSize: 15,
+    marginTop: 10,
+    marginBottom: 15,
+    marginLeft: 10,
+    borderBottomWidth: 2,
+    borderBottomColor: '#F0F1F6',
```

Comment:
Good practices demands to create a `themes.js` file where you define all colors used in the project.